skimage/transform/_warps_cy.pyx

```diff
@@ -2,8 +2,10 @@
 #cython: boundscheck=False
 #cython: nonecheck=False
 #cython: wraparound=False
-import numpy as np
-cimport numpy as cnp
+cimport numpy as np
+np.import_array()
```

Comment:
what does this do?

Reply:
I don't think this necessary as `cimport` should take care of it. I think..... otherwise, we have real issue in the library ^_^

Reply:
Well then it seems we do have a library issue, check [this](https://numpy.org/devdocs/reference/c-api/array.html#importing-the-api).

Reply:
Ah, we seldom use the capi directly, you seem to be using it.